rllib/examples/custom_logger.py

```diff
@@ -75,13 +76,12 @@ def flush(self):
 
     ray.init(num_cpus=args.num_cpus or None)
 
-    config = {
-        "env": "CartPole-v0" if args.run not in ["DDPG", "TD3"] else "Pendulum-v1",
-        # Use GPUs iff `RLLIB_NUM_GPUS` env var set to > 0.
-        "num_gpus": int(os.environ.get("RLLIB_NUM_GPUS", "0")),
-        "framework": args.framework,
+    config = (
+        AlgorithmConfig().environment(
+            "CartPole-v0" if args.run not in ["DDPG", "TD3"] else "Pendulum-v1"
+        )
         # Run with tracing enabled for tfe/tf2.
-        "eager_tracing": args.framework in ["tfe", "tf2"],
+        .framework(args.framework, eager_tracing=args.framework in ["tfe", "tf2"])
```

Comment:
maybe we can start removing `tfe` from examples?

Reply:
Done in a different PR:
https://github.com/ray-project/ray/pull/29755